cpp/src/aztec/plonk/proof_system/widgets/transition_widgets/create_dummy_transcript.hpp

```diff
@@ -1,109 +0,0 @@
-#include <plonk/composer/standard_composer.hpp>
```

Comment:
Nothing in this file was being used, so it seemed silly to refactor rather than delete.